js/views/userPage/Store.js

```diff
@@ -329,11 +331,21 @@ export default class extends BaseVw {
       (this._$listingsContainer = this.$('.js-listingsContainer'));
   }
 
+  get $shippingFilterContainer() {
```

Comment:
Should this, and the other gets, be using getCachedEl instead?

I assume you're just replicating what's in use, it's ok to update this to use getCachedEl. If you want to replace the other gets in this file with getCachedEl that would be nice.